disco_aws_automation/disco_aws.py

```diff
@@ -198,11 +198,11 @@ def create_userdata(self, hostclass, owner, is_spotinst=False):
             fixed_ip_hostclass['logforwarder'], ""
         )
         data["environment_name"] = self.vpc.environment_name
+        data["domain_name"] = self.hostclass_option_default(hostclass, "domain_name")
         data["owner"] = owner or getpass.getuser()
         data["credential_buckets"] = " ".join(self.vpc.get_credential_buckets(self._project_name))
-        data["zookeepers"] = "[\\\"{0}:2181\\\"]".format(
-            self._get_hostclass_ip_address(fixed_ip_hostclass['zookeeper'], "")
-        )
+        data["zookeepers"] = "[\\\"mhczookeeper-{0}.{1}:2181\\\"]".format(
```

Comment:
nitpick: do we want to save the port as well so that it can be referenced here: https://github.com/amplifylitco/asiaq_astro_config/pull/2319/files#diff-d47bab0bbaf9b9664c98f23797ee303dR18